permissions/plugin-coverity.yml

```diff
@@ -3,9 +3,10 @@ name: "coverity"
 paths:
 - "org/jenkins-ci/plugins/coverity"
 developers:
-- "frossi"
-- "kdang"
+- "gkillough"
+- "jrichard"
+- "akamen"
+- "bdsoss"
 - "shami"
 - "jbriggs"
-- "ahcho0819"
-- "oleg_nenashev"
```

Comment:
Fine with me. At some point I helped Franco Rossi to release the plugin, but it was 2 years ago or so.